admin/routes/service-manager.js

```diff
@@ -221,4 +221,14 @@ router.post('/annul-pupil-undo/:slug',
   serviceManagerController.postPupilAnnulmentUndo
 )
 
+router.get('/attendance-codes',
+  isAuthenticated([roles.serviceManager]),
+  serviceManagerController.getAttendanceCodes
+)
+
+router.post('/attendance-codes-submit',
```

Comment:
I usually use the same URL as the GET for the POST.  There is nothing wrong with this, just pointing out it is different from the GET req. 😄